app/controllers/account_controller.rb

```diff
@@ -170,9 +170,9 @@ def change_password
   private
 
   def logout_user
-    if User.current.logged?
+    if current_user.logged?
       cookies.delete OpenProject::Configuration['autologin_cookie_name']
-      Token.delete_all(["user_id = ? AND action = ?", User.current.id, 'autologin'])
+      Token.delete_all(["user_id = ? AND action = ?", current_user.id, 'autologin'])
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.